widgetry/src/widgets/mod.rs

```diff
@@ -767,9 +766,6 @@ impl Widget {
         }
     }
 
-    pub(crate) fn take_menu<T: 'static + Clone>(self) -> Menu<T> {
```

Comment:
In some Java haze, apparently I thought downcasting made more sense than just having two constructors. Good cleanup.